pkg/util/status/build_status.go

```diff
@@ -70,11 +70,20 @@ const (
 	ReasonMessageInvalidArtifactsMapping api.StepFailureMessage = "Invalid artifacts mapping specified."
 
 	// ReasonArtifactsFetchFailed is the reason associated with a failure to
-	// download specified scripts in the application image.
-	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchScriptsFailed"
+	// download specified artifacts in the application image.
+	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchArtifactsFailed"
 	// ReasonMessageArtifactsFetchFailed is the message associated with a failure to
-	// download specified scripts in the application image.
-	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch scripts specified scripts."
+	// download specified artifacts in the application image.
+	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch specified artifacts."
```

Comment:
[this is what constitutes artifacts - scripts](once https://github.com/openshift/source-to-image/blob/28109cfc72cffa557659cf2ea249c03bf14a812a/pkg/build/strategies/sti/sti.go#L374)
[but "artifacts is also used for any files the user specified to be copied"](https://github.com/openshift/source-to-image/blob/28109cfc72cffa557659cf2ea249c03bf14a812a/pkg/build/strategies/sti/sti.go#L275)